crawler/mergeData.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+with open('yelpData.json', 'r') as yelpfile:
+    yelpdata = json.load(yelpfile)
+
+with open('googleData.json', 'r') as googlefile:
+    googledata = json.load(googlefile)
+
+def checkDuplicated(yelpAddress, googleAddress):
```

Comment:
if you're only comparing address here, how about naming this 'isSameAddress' or something similar?